### PR TITLE
Approve users

### PR DESCRIPTION
**Pull request by lanre-ade**

### CODE REVIEW

File: api/tests/test_authentication.py [hunk 1]
```diff
@@ -162,10 +164,26 @@ def test_registration_without_github(self):
...
+
+        self.c.basic_credentials(self.user_dict['username'], self.user_dict['password'])
+        response = self.c.get('/author/login')
+        self.assertEquals(response.status_code, 401, 'user should not be allowed to log in')
+
     def test_login(self):
         response = self.basic_client.get('/author/login')
         content = json.loads(response.content)
-
+        print(content)
```
Comment (xkb) [inline api/tests/test_authentication.py]: Oops.

File: api/views/authentication.py [hunk 2]
```diff
@@ -53,7 +54,7 @@ def post(self, request):
             token, created = Token.objects.get_or_create(user=author.user)
             serializer = AuthorSerializer(author)
 
-            return Response({'token': token.key, 'author': serializer.data}, status=status.HTTP_201_CREATED)
```
Comment (xkb) [inline api/views/authentication.py:56]: Can you make this whole response 201? No need of anything else anymore.

Comment (lanre-ade) [inline api/views/authentication.py:56]: Sure

**Comment by xkb** [comment]
This is neat and clean solution :+1:  squash down commits before merging :smile:

**Pull request merged**